### ISSUE TITLE
Updates / Blog posts

### ISSUE THREAD

**Issue by bradfrost** [bradfrost/pl-website-eleventy#10]
Build out blog functionality (blog index page and blog detail page)

**Comment by QuincyGow** [comment]
I added "blog" to navigation.  Created md file to start building out page.  Would like some clarification on where I'm pulling the links to blog post.  Am I pulling from the old site?

**Comment by bradfrost** [comment]
> I added "blog" to navigation.

That's not necessary. It exists here: https://pattern-lab-eleventy.netlify.com/updates/ and is linked to in the footer, which is fine.

>  Would like some clarification on where I'm pulling the links to blog post. Am I pulling from the old site?

Blog posts don't exist yet, so to build feel free to just create a couple posts with lorem ipsum. 

The two templates that need created are: 
- a blog index page (which is [this page](https://pattern-lab-eleventy.netlify.com/updates)) that should list out all blog posts in reverse chronological order and links through to each individual 
- A blog detail page that contains the specific blog post.

Eleventy (and Hylia, the boilerplate that we started with) has this functionality built right in. So definitely use that as a reference: https://hylia.website/

**Comment by QuincyGow** [comment]
Created two blog post under docs.  Put on the Updates page.  The links in updates aren't linking to the docs.  Syntax issue maybe?  I tried a bunch of different options but nothing seemed to stick.

**Comment by QuincyGow** [comment]
I had this one working perfectly. I noticed the browser stopped updating with me so I killed the terminal and ran npm start. Now, I'm getting this error back and the template for Updates won't load or let me change the template. Trying to get to the bottom of it but would love some help.
<img width="985" alt="Screen Shot 2020-01-22 at 1 29 40 PM" src="https://user-images.githubusercontent.com/48933977/72924062-370af680-3d1e-11ea-9df8-0460d69b4164.png">
